lib/config/index.js

```diff
@@ -0,0 +1,46 @@
+module.exports = getConfig
+
+var statSync = require('fs').statSync
+var path = require('path')
+
+var defaultsDeep = require('lodash').defaultsDeep
+var series = require('async').series
+
+var accountConfig = require('./account')
+var assureFolders = require('./assure-folders')
+var couchDbConfig = require('./db/couchdb')
+var getDefaults = require('./defaults')
+var pouchDbConfig = require('./db/pouchdb')
+var storeConfig = require('./store')
+
+function getConfig (server, config, callback) {
+  defaultsDeep(config, getDefaults())
+
+  var adapter = config.PouchDB.preferredAdapters[0]
```

Comment:
Seems a bit of a significant change, and I think it conflicts with this line in the readme at present: https://github.com/hoodiehq/hoodie-server/pull/500/files#diff-04c6e90faac2675aa89e2176d2eec7d8R59 - specifically: "store.couchdb, store.PouchDB are set based on db option above".

Seems there's no db option anymore, and it requires the user to setup their own PouchDB instance prior to passing options `@hoodie/server`?